src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,11 @@
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="page-content">
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard movie={movie} key={movie.imdbId} />
```

Comment:
import MovieCard in this file
![Screenshot from 2023-09-07 13-55-18](https://github.com/mate-academy/react_movies-list-js/assets/33029864/d5ee335d-9941-4079-b3c8-af2a9403cff3)

Reply:
done